package.go

```diff
@@ -0,0 +1,24 @@
+package catfruits
+
+import "regexp"
+
+var DefaultPakcageFuncs = map[string]PackageFunc{
```

Comment:
__[gometalinter]__
* [golint] exported var DefaultPakcageFuncs should have comment or be unexported